contracts/connection/connection.go

```diff
@@ -28,6 +28,11 @@ type Client interface {
 	BalanceAt(ctx context.Context, contract common.Address, blockNumber *big.Int) (*big.Int, error)
 }
 
+const (
+	mainnet = "mainnet"
```

Comment:
Naming convention please. `ChainMainnet = Chain("mainnet")` and `ChainRopsten = Chain("ropsten")` with a new type defined `type Chain string`.